addons/mrp/static/src/scss/mrp_workorder_kanban.scss

```diff
@@ -1,8 +1,5 @@
 .o_kanban_dashboard{
     &.o_mrp_workorder_kanban {
-        .o_kanban_group:not(.o_column_folded) {
-            width: 400px + $o-kanban-group-padding;
-        }
```

Comment:
if I remember this CSS is not working (because it's overridden by other kanban view flex CSS), so looks ok to remove